hw2/03-count.js

```diff
@@ -1,3 +1,19 @@
-// Add your code here
+const keywordsInput = document.querySelector("#keyword_input");
+const sentences = document.querySelector("#sentences");
```

Comment:
You may want to consider using the getElementByID() instead of querySelector() since you are defining the id's anyway.

Reply:
Updated my code to use getElementByID instead of querySelector.